ksql-engine/src/test/java/io/confluent/ksql/physical/PhysicalPlanBuilderTest.java

```diff
@@ -488,10 +488,10 @@ public void shouldCheckSinkAndResultKeysDoNotMatch() {
     final String planText = queryMetadataList.get(1).getExecutionPlan();
     final String[] lines = planText.split("\n");
     assertThat(lines.length, equalTo(4));
-    assertThat(lines[0],
-        equalTo(" > [ REKEY ] | Schema: [ROWKEY STRING KEY, COL0 BIGINT, COL1 STRING, COL2 DOUBLE] "
+    assertThat(lines[1],
```

Comment:
How come previously the rekey came before the sink? I didn't notice any changes in this PR that would've reordered steps. Is this related to how `KsqlStructuredDataOutputNode#buildStream()` now returns `result.into(...)` rather than `result`?

Reply:
It makes sense for the rekey to happen before the sink - there's no point rekeying the stream after writing it out. What we had before was just a bad plan description.